npm/pkg/dataplane/policies/iptables-comments_linux.go

```diff
@@ -0,0 +1,202 @@
+package policies
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/npm/pkg/dataplane/ipsets"
+	"k8s.io/klog"
+)
+
+/*
+	Notes on commenting
+
+	v1 overall:
+		"[value]" means include if needed e.g. if a port is specified
+		- prefix:
+			- no to/from rules and not allowing external:
+				- allowed: "ALLOW-ALL"
+				- denied: "DROP-ALL"
+			- otherwise: drop the "ALL"
+		- suffix:
+			- no to/from rules or port rules and not allowing external:
+				- ingress: "-FROM-all-namespaces"
+				- egress: "-TO-all-namespaces"
+			- otherwise: "" (no suffix)
+		- append these to each other to form the whole comment:
+			prefix
+			[-cidrIPSetName]
+			[-AND-nsSelectorComment]
+			[-AND]
+			[-podSelectorComment]
+			[-protocolComment]
+			[-portComment]
+			-TO  (or "-FROM" if egress)
+			-targetSelectorComment
+			suffix
+
+	v2 overall for ACLs:
+		- prefix:
+			- allowed:
+				- no IPSets in SrcList/DstList (i.e. allow external): "ALLOW-ALL"
+				- otherwise:
+					- ingress: "ALLOW-FROM"
+					- egress: "ALLOW-TO"
+			- denied: replace "ALLOW" with "DROP"
+		- similar idea (think there are at most two non-namedPort ipsets e.g. ns selector and pod selector):
+			prefix
+			[-ipset1Name]
+			[-AND]
+			[-ipset2Name]
+			[-ON-protocolComment]
+			[-TO-namedPortIPSetName]
+			[-TO-portComment]
+		NOTE: can have none or only one of namedPort and port (range)
+
+	v2 for jumps to ingress/egress chains:
+		- prefix:
+			- ingress: "INGRESS"
+			- egress: "EGRESS"
+		- form:
+			INGRESS     (or "EGRESS-" if egress)
+			-POLICY
+			-policyKey
+			-TO         (or "-FROM" if egress)
+			[-podSelectorComment]   (or "all" if there are no pod selectors)
+			-IN-ns
+			-namespaceName
+
+	strings for protocol, ports, selectors:
+		protocol: just "name"
+
+		port (range):
+			- v1:
+				- for single port: PORT-x
+				- for namedport: PORT-name
+			- v2:
+				- for single port: PORT-x
+				- with endport: PORT-x:y
+				- in v2, namedports are specified as IPSets
+
+		selector:
+			"[!]" means optionally include "!" if the label is not included
+			- namespace selectors (only for v1):
+				- if there are no match expressions or labels:
+					all-namespaces
+				- otherwise:
+					ns-[!]label1-AND-ns-[!]label2...
+			- other w/out ns:
+				[!]label1-AND-[!]label2...
+			- other w/ ns:
+				[!]label1-AND-[!]label2...-AND-ns-[!]labelM-IN-ns-name
+*/
+
+func (networkPolicy *NPMNetworkPolicy) commentForJumpToIngress() string {
+	return networkPolicy.commentForJump(true)
```

Comment:
Can you change true/false to constant types for ingressJump or egressJump, that was it will be easier to read ?